### DISCUSSION TITLE
Enabling Wazuh after installation

### DISCUSSION THREAD

**Discussion by its-a-me-mario-the-og** [Security-Onion-Solutions/securityonion#10288] [Unsupported Versions]
Hi everyone! Just wanting to know if Wazuh can be enabled after installation if you chose not to enable it in the initial install? I chose not to enable it and want to start doing more EDR but don't want to redo the whole cluster. If the only way is to redo the whole install that is fine but wanted to check before doing that, thank you in advance!

**Comment by cm-ops** [comment]
Edit your `/opt/so/saltstack/local/pillar/global.sls` change `wazuh: 0` to `wazuh: 1` then run `so-checkin`.
